src/main/java/mate/academy/dao/MovieDao.java

```diff
@@ -10,4 +10,8 @@ public interface MovieDao {
     Optional<Movie> get(Long id);
 
     List<Movie> getAll();
+
+    boolean update(Movie movie);
```

Comment:
Consider changing the return type of the `update` method to `void` or `Movie` to better reflect the operation's outcome. Returning a `boolean` might not be necessary if the method is expected to always succeed or throw an exception otherwise.